baton/modules/echo/echo.go

```diff
@@ -9,7 +16,18 @@ func (e Echo) RunCommand(cmd string, body interface{}, resp chan<- interface{})
 	if cmd != "echo" {
 		return errors.New("unknown command: " + cmd)
 	}
-	//send resp <- msg
 	resp <- body
 	return nil
-}
\ No newline at end of file
+}
+
+func (e Echo) Handler() *mux.Router {
+	m := mux.NewRouter()
+	m.PathPrefix("/").HandlerFunc(echo)
+	return m
+}
+func echo(w http.ResponseWriter, r *http.Request) {
```

Comment:
Should there be an empty line between these two functions? Was `gofmt` run on this file?